tedana/workflows/t2smap.py

```diff
@@ -176,6 +187,8 @@ def t2smap_workflow(
     mask : :obj:`str`, optional
         Binary mask of voxels to include in TE Dependent ANAlysis. Must be spatially
         aligned with `data`.
+    masktype : {'dropout', 'decay'} or :obj:`list`, optional
```

Comment:
```suggestion
    masktype : :obj:`list` with 'dropout' and/or 'decay' or None, optional
```
I'm fairly sure the function requires a list, even for a single element so i reworded the docstring.